.changeset/kind-dryers-beam.md

```diff
@@ -0,0 +1,65 @@
+---
+'@makeswift/next-plugin': minor
+'@makeswift/runtime': minor
+---
+
+refactor: replaces the runtime draft/preview mode proxy endpoint with
+middleware, and removes draft/preview rewrite rules from the next config
+decorator.
+
+## Breaking Changes
+
+To upgrade, you'll need to add the new Makeswift middleware to your project. If
+you don't already have middleware for your site, create a `middleware.ts` file
+at the same level as where your `app/` or `pages/` directory is located.
+
+Then, use the new `withMakeswiftMiddleware` function to create a middleware and
+export the result. You will need to pass your API key:
+
+```ts
+import { MAKESWIFT_SITE_API_KEY } from '@/makeswift/env'
```

Comment:
Could we point to a place in the docs that shows where this value comes from? Otherwise readers might think this is some sort of special way to get the Makeswift API key